docs/blog/_posts/2019-04-05-14th-dotty-milestone-release.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.14.0-RC1 with export, immutable arrays, creator applications and more
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2019-04-05

```

Comment:
put here the date of the blogpost (estimation). lets say 10 (also the filename)

Reply:
Ok